src/metric-engine/src/metadata_region.rs

```diff
@@ -157,6 +157,24 @@ impl MetadataRegion {
 
         Ok(columns)
     }
+
+    pub async fn logical_regions(&self, physical_region_id: RegionId) -> Result<Vec<RegionId>> {
+        let metadata_region_id = utils::to_metadata_region_id(physical_region_id);
+        let region_prefix = "__region_".to_string();
```

Comment:
It's also used in `src/metric-engine/src/metadata_region.rs`.

I think we can extract a constant.